snuba/datasets/processors/spans_processor.py

```diff
@@ -224,165 +181,53 @@ def _process_measurements(
                 )
 
     @staticmethod
-    def _process_module_details(
+    def _process_sentry_tags(
```

Comment:
We would need to do the following as well. Do you want to merge this PR and handle this case later? Or do you want to do the migration of creating the column first and then handle the case in this PR itself?
1. Add `sentry_tags` column to the spans clickhouse cluster.
2. When 1 is done, remove all fields from `sentry_tags` which are promoted to top level columns.
3. Remaining `sentry_tags` need to be populated in the clickhouse `sentry_tags` column.

Reply:
Yes I'll handle in a later PR after a migration PR is done.